.github/workflows/ci.yml

```
@@ -0,0 +1,309 @@
+name: continuous-integration
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+    paths-ignore:
+      - '**.md'
+      - '**/**.md'
+      - '**/**/**.md'
+      - '**/**/**/**.md'
```

```suggestion
      - '**/*.md'
```

A bit too verbose. `'**/*.md'` Will do the same.